luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -0,0 +1,294 @@
+
+
+function gadget:GetInfo()
+	return {
+		name = "Team Power Watcher",
+		desc = "Tracks power of individual and total units per team. To be used for PvE dynamic difficulty and library functions to assertain aspects of game progression",
+		author = "SethDGamre",
+		date = "2024-07-12",
+		license = "None",
+		layer = 1,
+		enabled = true
+	}
+end
+
+if gadgetHandler:IsSyncedCode() then
```

Comment:
do `if not gadgetHandler:IsSyncedCode() then return end` and save an indentation level across the whole file

Reply:
Wait a minute, I wrote an identical comment but just realized it's not in the review. Must have forgotten to hit the button 🤷

Reply:
K, I'll do that.